### DISCUSSION TITLE
Is there any way to apply bulk edits to repository paths?

### DISCUSSION THREAD

**Discussion by danielrosehill** [alefragnani/vscode-project-manager#768] [Q&A] [answered]
I recently did some organisation of my local storage and created subfolders within `/home/git` to organise my repositories a little better. 

Unfortunately (as I should have thought beforehand) this broke the paths in the config file that the extension relies upon to hold the shortcuts.

But I would like to be able to move the repos as required...

Is there any way to allow for this possibility ? Maybe symlinks or something like that?

TY

**Comment by alefragnani** [comment, marked as answer]
Hi @danielrosehill ,

The extension does not have anything specifically to deal with this kind of reorganization, but depending on the kind of project you are working, you may have different alternatives.

### Favorite Projects

They are stored (today) in a JSON file, which you can easily open using the `Project Manager: Edit Projects` command. While opened, you may update the paths freely, using _multi-cursor editing_ to speed up the process, and once you save the file, the projects are automatically updated.

### Autodetected projects

Once you update the `projectManager.git.baseFolders` setting, you can simply run the `Project Manager: Refresh Git Projects`, and the project list will be automatically updated as well.

About symlinks, the extension does support. You just have to enable the `projectManager.supportSymlinksOnBaseFolders` setting. I’m not a fan of symlinks, but I suppose it would fit future reorganizations as well.

Hope this helps
